crates/infra/utils/src/cargo/workspace.rs

```diff
@@ -112,7 +112,13 @@ impl CargoWorkspace {
                 "--config",
                 format!(
                     "build.rustflags = {rustflags}",
-                    rustflags = serde_json::to_string(&["--deny", "warnings"])?,
+                    rustflags = serde_json::to_string(&[
+                        "--deny",
+                        "warnings",
```

Comment:
nit suggestion: to match:

```suggestion
                        // Deny any warnings in CI (turn into errors):
                        "-Dwarnings",
```

Reply:
Sure, done in cfba8525f000fd04efd9e25d41cb4ebb8f679014